mp/src/game/client/momentum/ui/SettingsPanel/HUDSettingsPanel.cpp

```diff
@@ -32,6 +32,7 @@ HUDSettingsPanel::HUDSettingsPanel(Panel *pParent, Button *pAssociate) : BaseCla
     m_pSpeedometerType->AddItem("#MOM_Settings_Speedometer_Type_Absolute", nullptr);
     m_pSpeedometerType->AddItem("#MOM_Settings_Speedometer_Type_Horiz", nullptr);
     m_pSpeedometerType->AddItem("#MOM_Settings_Speedometer_Type_Vert", nullptr);
+    m_pSpeedometerType->AddItem("#MOM_Settings_Speedometer_Type_ExplosiveJump", nullptr);
```

Comment:
Something to be wary about in the future is adding to the middle of things, if existing speedometer.vdf files map enum type / array index to speedometers, you'll have an off-by-one when inserting into the middle. It's not a problem here due to this speedometer refactor happening this update, but something to keep in mind for the future.

Reply:
And on further reading it seems that order is determined by speedometer name and not any integer or something, so that's also good. But for something like gamemode, you'd want to add to the end of the enum / array.